src/spikeinterface/core/segmentutils.py

```diff
@@ -18,13 +18,13 @@ def _check_sampling_frequencies(sampling_frequency_list, sampling_frequency_max_
             f"`sampling_frequency_max_diff`={sampling_frequency_max_diff}Hz"
         )
     elif max_diff > 0:
-        diff_sec = 24 * 3600 * max_diff / freq_0
+        diff_msec = 24 * 3600000 * max_diff / freq_0
```

Comment:
```suggestion
        diff_ms = 24 * 3600000 * max_diff / freq_0
```
We use `ms` all over the place ;)